tests/test_simpleTable.py

```diff
@@ -271,6 +271,14 @@ def testColumnView(self):
             self.assertEqual(catalog[i].get(kI), 4)
             self.assertEqual(catalog[i].get(kD), f3v[i])
 
+        # Accessing an invalid key should raise.
+        for keyType in ["Angle", "ArrayB", "ArrayD", "ArrayF", "ArrayI",
+                        "ArrayU", "B", "D", "F", "Flag", "I", "L", "U"]:
+            # Default-constructed key is invalid
+            invalidKey = getattr(lsst.afw.table, f"Key{keyType}")()
+            self.assertFalse(invalidKey.isValid())
+            self.assertRaises(lsst.pex.exceptions.LogicError, catalog.get, invalidKey)
```

Comment:
I find the context manager version easier to understand the intent:

```
with self.assertRaises(LogicError):
    catalog.get(invalidKey)
```

Reply:
The testing guide prefers the context manager. Currently about 90% of the LSST code bases complies with that statement.

https://developer.lsst.io/python/testing.html?highlight=assertraises